frontend/copr-frontend.spec

```diff
@@ -296,16 +296,22 @@ EOF
 %py_byte_compile %{__python3} %{buildroot}%{_datadir}/copr/coprs_frontend/coprs
 %py_byte_compile %{__python3} %{buildroot}%{_datadir}/copr/coprs_frontend/alembic
 
+%if 0%{?fedora} && 0%{?fedora} > 41
+install -m0644 -D conf/copr-frontend.sysusers.conf %{buildroot}%{_sysusersdir}/copr-frontend.conf
+%endif
+
 %check
 %if %{with check}
 ./run_tests.sh -vv --no-cov
 %endif
 
+%if 0%{?fedora} && 0%{?fedora} > 41
```

Comment:
did you mean 0%{?fedora} < 41 perhaps? since this way we will end without copr-fe user and group for f41 and less

Reply:
no. because for fedora <41 the old behaviour will be used.